test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml

```diff
@@ -121,6 +121,10 @@ tests:
         values: '1+1x780'
       - series: 'taskrun_pod_create_not_attempted_or_pending_count{namespace="tenant-2", source_cluster="cluster01"}'
         values: '0x780'
+      - series: 'tekton_pipelines_controller_running_taskruns_throttled_by_quota{namespace="tenant-1", source_cluster="cluster01"}'
+        values: '0x780'
+      - series: 'tekton_pipelines_controller_running_taskruns_throttled_by_node{namespace="tenant-1", source_cluster="cluster01"}'
+        values: '0x780'
```

Comment:
Shouldn't there be some tests where these time series have values other than 0?

Reply:
yep ... too much multi-tasking ... thanks

Reply:
additional tests added @amisstea thanks